tutorials/Tutorial 8 - Directed Hypergraphs.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    print("Head and tail of edge 0:")
</pre><p>Do we want to keep the "head" and "tail" terminology or go for the other one? I'm wonder if we should really allow both, might get confusing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Let's go with head and tail for now since that is the standard terminology, but senders and receivers might be more explainable in the future. For that reason, I'm keeping the comment above as is.</p>